crates/networking/rpc/based/env.rs

```diff
@@ -0,0 +1,97 @@
+use ethrex_common::{Address, H256, U256};
+use serde::{Deserialize, Serialize};
+use ssz_types::{typenum, VariableList};
+use tree_hash::TreeHash;
+
+use crate::{utils::RpcErr, RpcApiContext};
+
+pub type MaxExtraDataSize = typenum::U256;
+pub type ExtraData = VariableList<u8, MaxExtraDataSize>;
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct EnvV0 {
+    pub number: u64,
+    pub parent_hash: H256,
+    pub beneficiary: Address,
+    pub timestamp: u64,
+    pub gas_limit: u64,
+    #[serde(rename = "baseFee")]
+    pub basefee: u64,
+    pub difficulty: U256,
+    pub prevrandao: H256,
+    #[serde(with = "ssz_types::serde_utils::hex_var_list")]
+    pub extra_data: ExtraData,
+    pub parent_beacon_block_root: H256,
+}
+
+impl EnvV0 {
+    pub fn handle(&self, _context: RpcApiContext) -> Result<serde_json::Value, RpcErr> {
+        tracing::info!("handling env");
```

Comment:
Do we really need this log? Maybe this can have debug level?

Reply:
Changed this and other info logs with debug logs [ac26560](https://github.com/lambdaclass/ethrex/pull/2274/commits/ac2656018100cc9d2fe5592390c091611a014ebf).